### ISSUE TITLE
Debian package scripts fail to update Prosody configuration files

### ISSUE THREAD

**Issue by paulmenzel** [jitsi/jitsi-meet#10282]
I experienced the same issue as reported in the community forum thread [*Jicofo needs restart because of “Failed to create room: JvbBrewery@internal.auth.mydomain.com” after system boot*][1].

I am on Debian 10, so Prosody 0.11.x has been used from the beginning. (*prosody* 0.11.2-1 installed on March 18th, 2020.)

    $ dpkg -l prosody | grep ^i
    ii  prosody        0.11.2-1+deb10u2 amd64        Lightweight Jabber/XMPP server

It looks like the Prosody configuration in `/etc/prosody/conf.d/` is not updated when upgrading the packages.

[1]: https://community.jitsi.org/t/jicofo-needs-restart-because-of-failed-to-create-room-jvbbrewery-internal-auth-mydomain-com-after-system-boot/65447

**Comment by damencho** [comment]
The correct domains are used in the script, you mean that the debian package added `jitmeet.example.com` as your domain?

**Comment by damencho** [comment]
Aaa I see a previous update broke it ... and now that is fixed. I mean if you skip the broken update everything will be alright?

**Comment by paulmenzel** [comment]
No, I had to fix it manually, as I already had installed the broken update adding the `admin =` entry, but without the other two lines it wasn’t a problem (unused?). (Probably the differences between 2020 and 2021 are easily missed in my comment.) Anyway, once the merge/pull request is accepted, then users with old installations hopefully won’t run into the issue. I just wanted to check, how I got into the broken state, and thanks to [etckeeper](https://etckeeper.branchable.com/), it can be easily traced back.
